app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id', dependent: :destroy
-  has_many :comments, dependent: :destroy
-  has_many :likes, dependent: :destroy
```

Comment:
Dear @harshi0102 
Good job for adding the user's relation with posts However, there're two relations that are missing. As you can see in the ERD diagram. The user also has many Likes and User has many Comments. Kindly implement that as well 👍 

<img width="823" alt="Screenshot 2023-07-19 at 10 21 46 PM" src="https://github.com/harshi0102/blog-app-rails/assets/83048208/4a2a89da-3a4f-4943-b608-c6f1886873c1">